core/app/c/[communitySlug]/pubs/[pubId]/page.tsx

```diff
@@ -154,9 +153,6 @@ export default async function Page({
 							) : null}
 						</div>
 					</div>
-					<div>
-						<MembersAvatars pub={pub} />
-					</div>
```

Comment:
This wasn't rendering anything on most pubs, and it's confusing when it is rendered. See the "Assigned To" section  on this [pub](https://app.pubpub.org/c/unjournal/pubs/3997773c-d691-415e-a0c4-d4b4f7de29ac)
![image](https://github.com/user-attachments/assets/57e89150-71af-49af-9c4b-389ed8f05370)